README.md

```diff
@@ -102,156 +103,127 @@ There are [community driven install methods](./doc/unofficialInstallMethods.md)
+- Refactored service with cleaner state management for stability.
 
 ### Mouse Pointer Crosshairs
- - Added Esc key to cancel active gliding cursor sequence. Thanks [@mikehall-ms](https://github.com/mikehall-ms)!
- - Added orientation option (vertical / horizontal / both) for crosshairs customization. Thanks [@mikehall-ms](https://github.com/mikehall-ms)!
+ - Enabled switching between Mouse Pointer Crosshairs and Gliding Cursor modes. Thanks [@mikehall-ms](https://github.com/mikehall-ms).
 
 ### Mouse Without Borders
- - Continued Common class refactor (part 5/7) by extracting clipboard and init/cleanup logic into focused classes. Thanks [@mikeclayton](https://github.com/mikeclayton)!
-
- - Fix connection failures caused by conflicting MachineId across machines.  Thanks [@noraa-junker](https://github.com/noraa-junker) for troubleshooting!
+ - Added horizontal scrolling support. Thanks [@MasonBergstrom](https://github.com/MasonBergstrom).
 
 ### Peek
- - Added the option to activate Peek with just the Spacebar.
+- Fixed media files remaining locked after preview window closes. Thanks [@daverayment](https://github.com/daverayment)!
+- Added a command-line interface for file previewing. See the [Peek documentation](https://learn.microsoft.com/en-us/windows/powertoys/peek) for usage. Thanks [@prochan2](https://github.com/prochan2)!
 
 ### PowerRename
- - Fixed enumeration counter skipping when regex replacement equals original filename (counters now advance reliably). Thanks [@daverayment](https://github.com/daverayment)!
+- PowerRename no longer crashes due to a missing resources file.
+- Added photo metadata extraction support using EXIF and XMP for pattern-based renaming with camera info, GPS coordinates, and date taken.
 
-### Quick Accent
- - Expanded Welsh layout with acute, grave, and dieresis variants for vowels (consistent ordering). Thanks [@PesBandi](https://github.com/PesBandi)!
+### PowerToys Run
+ - Added retry logic with exponential backoff to handle DWM composition errors during theme changes. Thanks [@jiripolasek](https://github.com/jiripolasek).
+ - Update OneNote png icons to new core10 design. Thanks [@trevorNgo](https://github.com/trevorNgo).
 
-### Registry Preview
- - Migrated to native TitleBar and AppWindow APIs for cleaner window chrome.
+ ### Quick Accent
+ - Added diameter symbol (⌀) for Shift+O in Special Characters mode, thanks to [@anselumjuju](https://github.com/anselumjuju).
 
-### Screen Ruler
- - Fixed ARM64 crash by aligning cursor position structure to 8-byte boundary.
+### Zoomit
+ - Smooth out zoom-animation in ZoomIt by coalescing mouse-move and timer events, thanks to [@foxmsft](https://github.com/foxmsft).
+ - Enable GIF support for ZoomIt, thanks to [@MarioHewardt](https://github.com/MarioHewardt).
+ - Fix spelling mistakes, and refactored some literal strings to string constants, thanks to [@lzandman](https://github.com/lzandman).
 
```

Comment:
I fixed a GDI object leak and a bug with Actual Size screenshots for ZoomIt in #43375.